app/scripts/modules/azure/loadBalancer/loadBalancer.transformer.js

```diff
@@ -146,24 +146,48 @@ module.exports = angular.module('spinnaker.azure.loadBalancer.transformer', [
         detail: 'frontend',
         credentials: defaultCredentials,
         region: defaultRegion,
-        vpcId: null,
-        healthCheckProtocol: 'HTTP',
-        healthCheckPort: 7001,
-        healthCheckPath: '/healthcheck',
-        healthTimeout: 5,
-        healthInterval: 10,
-        healthyThreshold: 10,
-        unhealthyThreshold: 2,
-        regionZones: [],
+        cloudProvider: 'azure',
+        vnet: null,
+        probes: [
+          {
+            probeName: '',
+            probeProtocol: 'HTTP',
+            probePort: 7001,
+            probePath: '/healthcheck',
+            probeInterval: 10,
+            unhealthyThreshold: 2
+          }
+        ],
         securityGroups: [],
-        listeners: [
+        loadBalancingRules: [
           {
-            internalProtocol: 'HTTP',
-            internalPort: 7001,
-            externalProtocol: 'HTTP',
-            externalPort: 80
+            ruleName: '',
+            protocol: 'TCP',
+            externalPort: 80,
+            backendPort: 8080,
+            probeName: '',
+            persistence: 'None',
+            idleTimeout: 4,
           }
-        ]
+        ],
+        name: '',
+        user: 'anonymous',
+        stageDetails: {
+          name: null,
+          type: 'upsertLoadBalancer',
+          startTime: '',
+          isSynthetic: false,
+        },
+        batch:
+        {
+          task:
+          {
+            id:
+            {
+              stageStart: 41,
+            },
+          },
+        },
```

Comment:
These lines seem like they shouldn't be there. `name` and `user` should be populated by the write service; `stageDetails` and `batch` seem like they are fields we'd get back from the executing task.